### PR TITLE
NMS-15029: Update QS Visualize chapter

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `operation/deep-dive/admin/webui/opsboard/introduction.adoc`
- Update `operation/deep-dive/admin/webui/surveillance-views.adoc`
- Update `operation/quick-start/visualize-data.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-15029

### CODE REVIEW

File: docs/modules/operation/pages/quick-start/visualize-data.adoc [hunk 1]
```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
...
+
+The Surveillance View and its corresponding dashboard component visualize all routers, switches, and servers in your production, test, and development environments by default (see <<deep-dive/admin/webui/surveillance-view.adoc#surveillance-view-default-config, Default configuration>>).
+You can create custom surveillance categories to define specific data that you want to visualize in the Surveillance View.
+Follow these steps to create a new surveillance category:
+
+. Click the *gear* symbol in the top-right of the screen.
+. Under Provisioning, click *Manage Surveillance Categories*.
+. Enter a name for the new category in the *Category Name* box, and click *Add New Category*.
+. Add nodes to the category as desired.
+** Choose nodes from the *Available Nodes* list and click *Add*.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/quick-start/visualize-data.adoc:55]: Step missing before "choose nodes ..."
. Click the *edit* symbol beside the new category.

File: docs/modules/operation/pages/quick-start/visualize-data.adoc [hunk 2]
```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
...
++
+A list of configured Surveillance Views is displayed.
+
+. Click *Add*, and define the view's title and the refresh timing, in seconds.
+. Add rows and columns to the view as desired:
+** To add rows, click *Add* beside the *Rows* list, type a name for the new row, and select its category from the *Category* list.
+Click *Save*.
+** To add columns, click *Add* beside the *Columns* list, type a name for the new column, and select its category from the *Category* list.
+Click *Save*.
+. Click *Save* to create the new Surveillance View.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/quick-start/visualize-data.adoc:72]: Add steps:

. Click *Preview* to see the surveillance view you just created.
. (Optional) Place a check in the box in the *Default* box to make this the view that appears by default. 

Note that the Surveillance View in the dashboard includes a menu to select which view to display.

File: docs/modules/operation/pages/quick-start/visualize-data.adoc [hunk 3]
```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
...
 Just like in the Dashboard, you can select rows, columns, and cells to narrow your data focus.
 
-For more information, see xref:operation:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.
+For more information, see xref:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.
+
+=== Visualize data using surveillance categories
+
+The Surveillance View and its corresponding dashboard component visualize all routers, switches, and servers in your production, test, and development environments by default (see <<deep-dive/admin/webui/surveillance-view.adoc#surveillance-view-default-config, Default configuration>>).
+You can create custom surveillance categories to define specific data that you want to visualize in the Surveillance View.
+Follow these steps to create a new surveillance category:
```
Comment (Bonrob2) [inline docs/modules/operation/pages/quick-start/visualize-data.adoc:49]: Could we be more specific? Like a surveillance category that monitors X?

File: docs/modules/operation/pages/quick-start/visualize-data.adoc [hunk 4]
```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
...
+
+. Click the *gear* symbol in the top-right of the screen.
+. Under Provisioning, click *Manage Surveillance Categories*.
+. Enter a name for the new category in the *Category Name* box, and click *Add New Category*.
+. Add nodes to the category as desired.
+** Choose nodes from the *Available Nodes* list and click *Add*.
+
+Your changes are automatically saved.
+
+Once you have created a custom surveillance category, you can create a new Surveillance View to update the data that is visualized:
```
Comment (Bonrob2) [inline docs/modules/operation/pages/quick-start/visualize-data.adoc:59]: Could this also include a specific example? Probably so they can select the category that they created in the previous section.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Some additions and updates. Thanks!

File: docs/modules/operation/pages/quick-start/visualize-data.adoc [hunk 5]
```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
...
+
+=== Visualize data using surveillance categories
+
+The Surveillance View and its corresponding dashboard component visualize all routers, switches, and servers in your production, test, and development environments by default (see <<deep-dive/admin/webui/surveillance-view.adoc#surveillance-view-default-config, Default configuration>>).
+You can create custom surveillance categories to define specific data that you want to visualize in the Surveillance View.
+Follow these steps to create a new surveillance category:
+
+. Click the *gear* symbol in the top-right of the screen.
+. Under Provisioning, click *Manage Surveillance Categories*.
+. Enter a name for the new category in the *Category Name* box, and click *Add New Category*.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/visualize-data.adoc:53]:
```suggestion
. Enter a name for the new category in the *Category Name* box, and click *Add New Category*.
. Click the *edit* symbol beside the new category.
```

File: docs/modules/operation/pages/quick-start/visualize-data.adoc (hunk 2 above)
Comment (emarsh1995) [reply docs/modules/operation/pages/quick-start/visualize-data.adoc:72]:
```suggestion
. Click *Save* to create the new Surveillance View.
. Click *Preview* to see the Surveillance View that you just created.
. (Optional) Select *Default* beside the new Surveillance View to make it the one that appears by default.

NOTE: The Surveillance View dashboard component also includes a menu where you can select the view to be displayed.
```

File: docs/modules/operation/pages/deep-dive/admin/webui/opsboard/introduction.adoc [hunk 6]
```diff
@@ -26,8 +26,27 @@ image::webui/opsboard/02_opsboard-concept.png[]
...
+. Under Additional Tools, click *Ops Board Configuration*.
+. Click the *plus* symbol (*+*) beside the Overview tab, type a name for the new ops board, and click *Save*.
+. Click *Add Dashlet*, select a dashlet from the *Dashlet* list, and configure its settings:
+** *Title:* dashlet name to display in the ops board.
+** *Priority:* how often the dashlet displays in the rotation.
+Priority 1 is the highest, meaning it appears the most often.
+** *Duration:* how long the dashlet displays in the rotation, in seconds.
+** *Boost-Priority:* change the dashlet's priority if it is in the alert state.
+This setting is optional, and is not available for all dashlets.
+** *Boost-Duration:* change the display duration if the dashlet is in the alert state.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/admin/webui/opsboard/introduction.adoc:41]:
```suggestion
** *Boost-Duration:* change the display duration if the dashlet is in an alert state.
```

File: docs/modules/operation/pages/deep-dive/admin/webui/opsboard/introduction.adoc [hunk 7]
```diff
@@ -26,8 +26,27 @@ image::webui/opsboard/02_opsboard-concept.png[]
...
+
+. Click the *gear* symbol at the top-right of the screen.
+. Under Additional Tools, click *Ops Board Configuration*.
+. Click the *plus* symbol (*+*) beside the Overview tab, type a name for the new ops board, and click *Save*.
+. Click *Add Dashlet*, select a dashlet from the *Dashlet* list, and configure its settings:
+** *Title:* dashlet name to display in the ops board.
+** *Priority:* how often the dashlet displays in the rotation.
+Priority 1 is the highest, meaning it appears the most often.
+** *Duration:* how long the dashlet displays in the rotation, in seconds.
+** *Boost-Priority:* change the dashlet's priority if it is in the alert state.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/admin/webui/opsboard/introduction.adoc:39]:
```suggestion
** *Boost-Priority:* change the dashlet's priority if it is in analert state.
```

**Review by mmahacek — APPROVED** [review]
LGTM

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**